packages/eas-cli/src/commands/workflow/logs.ts

```diff
@@ -0,0 +1,172 @@
+import { Flags } from '@oclif/core';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import { EASNonInteractiveFlag, EasJsonOnlyFlag } from '../../commandUtils/flags';
+import {
+  WorkflowJobResult,
+  WorkflowLogLine,
+  processLogsFromJobAsync,
+} from '../../commandUtils/workflows';
+import { AppQuery } from '../../graphql/queries/AppQuery';
+import { WorkflowJobQuery } from '../../graphql/queries/WorkflowJobQuery';
+import { WorkflowRunQuery } from '../../graphql/queries/WorkflowRunQuery';
+import Log from '../../log';
+import { promptAsync } from '../../prompts';
+import formatFields from '../../utils/formatFields';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+
+export default class WorkflowView extends EasCommand {
+  static override description =
+    'view logs for a workflow run, selecting a job and step to view. You can pass in either a workflow run ID or a job ID. If no ID is passed in, you will be prompted to select from recent workflow runs for the current project.';
+
+  static override flags = {
+    ...EasJsonOnlyFlag,
+    ...EASNonInteractiveFlag,
+    allSteps: Flags.boolean({
```

Comment:
Looking at e.g. `build`, I think `eas-cli` tends to use `kebab-case-for-flags`. Are you thinking of changing this convention?

Reply:
No I can use kebab case, no problem